packages/ui/src/app/pages/Proposals/PastProposals.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+import { AppPage } from '@/app/components/AppPage'
+import { ActivitiesBlock } from '@/common/components/Activities/ActivitiesBlock'
+import { Loading } from '@/common/components/Loading'
+import { ContentWithSidepanel, MainPanel, SidePanel } from '@/common/components/page/PageContent'
+import { PageHeader } from '@/common/components/page/PageHeader'
+import { PageTitle } from '@/common/components/page/PageTitle'
+import { useActivities } from '@/common/hooks/useActivities'
+import { AddProposalButton } from '@/proposals/components/AddProposalButton'
+import { NoProposals } from '@/proposals/components/NoProposals'
+import { ProposalList } from '@/proposals/components/ProposalList'
+import { usePastProposals } from '@/proposals/hooks/usePastProposals'
+
+import { ProposalsTabs } from './components/ProposalsTabs'
+
+export const PastProposals = () => {
+  const { proposals, isLoading } = usePastProposals()
+  const activities = useActivities()
+
+  return (
+    <AppPage>
+      <PageHeader>
+        <PageTitle>Proposals</PageTitle>
+        <AddProposalButton />
+        <ProposalsTabs />
+      </PageHeader>
+
+      {isLoading ? (
+        <MainPanel>
+          <Loading />
+        </MainPanel>
+      ) : proposals.length ? (
+        <ContentWithSidepanel>
+          <MainPanel>
+            <ProposalList proposals={proposals} isPast />
+          </MainPanel>
+          <SidePanel>
+            <ActivitiesBlock activities={activities} label="Proposals Activities" />
+          </SidePanel>
+        </ContentWithSidepanel>
+      ) : (
+        <MainPanel>
```

Comment:
One thing – I think that we could have activities and proposals load independently. So maybe we should start working on this sooner? WDYT?

ATM activities are static (since we're waiting for the schema upgrade) but will be also loaded from GraphQL.

I would go with this layout:

```tsx
<ContentWithSidePanel>
  <MainPanel>{isLoading ? <Loadaing /> : ... }</MainPanel>
  <SidePanel><ActivitiesBlock ... /></SidePanel />
</ContentWithSidePanel />
```

Reply:
Having this we will display something while the other thins will be loading.

Activities will remain static for now :man\_shrugging:

Reply:
You're right I got confused because of the no current proposals case from the wiredframe. But I think even for the current proposal the sidebar could remain until we know there's no current proposals. WDYT ?

Reply:
Yeah, exactly :+1:

Reply:
Yeah, exactly :+1: